.github/workflows/e2e_tests.yml

```diff
@@ -7,6 +7,16 @@ on:
     branches: [main]
   schedule:
     - cron: '0 0 * * *'
+  workflow_dispatch:
+    inputs:
+      network:
+        description: 'Network'
+        required: false
```

Comment:
I think the `description` can be better. Perhaps:
`Select the blockchain network for testing.` or something better.